### DISCUSSION TITLE
The right camera for me

### DISCUSSION THREAD

**Discussion by MasterOfPuppets111** [blakeblackshear/frigate#10214] [General]
Hello folks,

I currently have 3 Annke NC800 cameras in use in conjunction with Frigate. I'm very happy with these so far. In addition, I would like to have 2 more cameras with the following features:
-PTZ
-2-way audio
-Color night vision
-Autotracking (via Frigate)
-IR lighting permanently at night, switching to color night vision when an object is detected
-4K 8MP

What bothers me a little with my NC800 is that the LEDs are on all night, so basic lighting around the house is not bad, but it is very noticeable from the outside.

That's why I first came across the Reolink RLC-823A, which I actually really like at first glance and has exactly what I'm looking for:
-4k 8MP
-PTZ
-2-way audio
-At night in IR mode, switching to spotlight with tracking when object is detected

What I don't like about this camera is the loud noise in the recordings during tilt & pan, but I could live with that. However, what is not possible with this camera at all is zooming during tracking, as this is an optical zoom and it takes a few seconds until the focus is sharp again.

Do you know a camera with the following features:

-PTZ
-4k 8MP
-2-way audio
-At night in IR mode, switching to spotlight with tracking when object is detected
-Zoom while tracking
-full Onif functionality so I can leave the tracking to frigate
-little engine noise in the recordings

If not, I would use the RLC-823A and forego zooming during tracking, but the question is whether frigate can take over autotracking with this model.

greetings pierre

**Comment by NickM-27** [comment]
> That's why I first came across the Reolink RLC-823A, which I actually really like at first glance and has exactly what I'm looking for:

these cameras have small sensors for 4k resolution. You should look for a camera with a larger sensor if night detection is important to you
